src/current/_includes/releases/v25.1/v25.1.8.md

```diff
@@ -0,0 +1,53 @@
+ [#146509][#146509]
+- Fixed a bug where the `kv.rangefeed.closed_timestamp.slow_ranges` would not be incremented when a rangefeed closed timestamp was slower than the target threshold.
+ [#146973][#146973]
+- Fixed a bug that caused the SQL Activity > Statement Fingerprint page to fail to load details for statements run with application names containing a `#` character.
+ [#147222][#147222]
+- Fixed a bug that could cause an `AFTER` trigger to fail with `client already committed or rolled back the transaction` if the query also contained foreign-key cascades. The bug had existed since `AFTER` triggers were introduced in v24.3.
+ [#147310][#147310]
+- Fixed a bug that could cause the `cockroach` process to `segfault` when collecting runtime execution traces (typically collected via the **Advanced Debug** page in the Console).
+ [#147332][#147332]
+- Previously, CockroachDB could incorrectly evaluate the `to_regclass`, `to_regnamespace`, `to_regproc`, `to_regprocedure`, `to_regrole`, and `to_regtype` built-in functions when the query using them was evaluated in a distributed fashion. The bug was introduced with these built-in functions in v23.1 and is now fixed.
+ [#147375][#147375]
+- Fixed a bug that caused the optimizer to ignore index hints when optimizing some forms of prepared statements. This could result in one of two unexpected behaviors: a query errors with the message `index cannot be used for this query` when the index can actually be used; or a query uses an index that does not adhere to the hint. The hints relevant to this bug are regular index hints, e.g., `SELECT * FROM tab@index`, `FORCE_INVERTED_INDEX`, and `FORCE_ZIGZAG`.
+ [#147416][#147416]
+- Fixed a bug that could cause stable expressions to be folded in cached query plans. The bug could cause stable expressions like `current_setting` to return the wrong result if used in a prepared statement. The bug was introduced in v23.2.22, v24.1.14, v24.3.9, v25.1.2, and the v25.2 alpha.
+ [#147459][#147459]
+- Fixed a bug where prepared statements on schema changes could fail with runtime errors.
+ [#147670][#147670]
+- Fixed a bug where `ALTER TABLE` was modifying identity attributes on columns not backed by a sequence.
+ [#147710][#147710]
+
+<h3 id="v25-1-8-performance-improvements">Performance improvements</h3>
+
+- TTL jobs now respond to cluster topology changes by restarting and rebalancing across available nodes.
+ [#147208][#147208]
+
+<h3 id="v25-1-8-miscellaneous">Miscellaneous</h3>
+
+- fixes an issue with LDR where the presence of a unique
+  index may cause spurious DLQ entries if the unique index has a smaller
+  index id than the primary key index. [#147353][#147353]
```

Comment:
You can change this note to: 

> Fixed an issue with logical data replication (LDR) where the presence of a unique index could have caused spurious dead-letter queue (DLQ) entries if the unique index had a smaller index ID than the primary key index.

I would delete the misc section and put this release note in bug fixes. Thanks!